packages/main/src/List.ts

```diff
@@ -352,6 +352,22 @@ class List extends UI5Element {
 	@property({ type: ListGrowingMode, defaultValue: ListGrowingMode.None })
 	growing!: `${ListGrowingMode}`;
 
+	/**
+	 * Defines the text that will be displayed inside the growing button at the bottom of the list,
```

Comment:
I think it is enough to say "Defines the text that will be displayed inside the growing button"